libs/ui/widgets/src/lib/popup-alert/popup-alert.component.stories.ts

```diff
@@ -35,19 +35,20 @@ export default {
 
 type PopupAlertComponentWithContent = PopupAlertComponent & { content: string }
 
+const content =
+  'Something went wrong during a task that was probably too complicated, you should probably <a href="https://www.google.com">look for answers</a> and come back!'
+
 const Template: Story<PopupAlertComponentWithContent> = (
   args: PopupAlertComponentWithContent
 ) => ({
   component: PopupAlertComponent,
   props: args,
-  template: `<gn-ui-popup-alert [icon]='icon' [position]='position'>${args.content}</gn-ui-popup-alert>`,
+  template: `<gn-ui-popup-alert [icon]='icon' [position]='position'>${content}</gn-ui-popup-alert>`,
```

Comment:
Hmm, shame that we're not able to change the content of the popup anymore :/

Reply:
content is not an input so it doesn't compile